build/deploy_ios_webapp.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# This script does not execute anything for now, but shows steps to deploy the application
+
+# 1. Create the docker image and push it to AWS ECR.  This image will process requests:
+#   > docker build -t 469205354006.dkr.ecr.us-east-1.amazonaws.com/fsoi-batch .
```

Comment:
How do you generate the image name (tag)?

Reply:
The number on the front is just the AWS Account ID: [account_id].dkr.ecr.us-east-1.amazonaws.com/[your_image_name].  I wonder if it would work to substitute 'jcsda' for the account ID, since that alias is in place.

Reply:
Oh - thanks - I thought it was some date-specific thing